dspace-api/src/main/java/org/dspace/storage/bitstore/S3BitStoreService.java

```diff
@@ -325,8 +345,28 @@ public Map about(Bitstream bitstream, Map attrs) throws IOException {
         try {
             ObjectMetadata objectMetadata = s3Service.getObjectMetadata(bucketName, key);
             if (objectMetadata != null) {
-                return this.about(objectMetadata, attrs);
+                if (attrs.containsKey("size_bytes")) {
+                    attrs.put("size_bytes", objectMetadata.getContentLength());
```

Comment:
Could we add tests for the changes to this `about()` method?  Codecov is saying that none of the newly added code in this method is tested.